packages/errors/src/error-codes.ts

```diff
@@ -78,6 +78,7 @@ export enum ErrorCode {
   TRANSACTION_SQUEEZED_OUT = 'transaction-squeezed-out',
   CONTRACT_SIZE_EXCEEDS_LIMIT = 'contract-size-exceeds-limit',
   MAX_INPUTS_EXCEEDED = 'max-inputs-exceeded',
+  MAX_OUTPUTS_EXCEEDED = 'max-outputs-exceeded',
```

Comment:
Could we add this to [`errors/index.md`](https://github.com/FuelLabs/fuels-ts/blob/master/apps/docs/src/guide/errors/index.md) please? :)

Reply:
Thanks for the remainder @petertonysmith94 https://github.com/FuelLabs/fuels-ts/pull/2899/commits/d971a07c06881ce0910b481300993d8076958acf